backend/src/main/java/ai/verta/modeldb/metadata/MetadataServiceImpl.java

```diff
@@ -1,237 +0,0 @@
-    } catch (Exception e) {
-      CommonUtils.observeError(
-          responseObserver, e, AddKeyValuePropertiesRequest.Response.getDefaultInstance());
-    }
-  }
-
-  @Override
-  public void getKeyValueProperties(
-      GetKeyValuePropertiesRequest request,
-      StreamObserver<GetKeyValuePropertiesRequest.Response> responseObserver) {
-    try {
-      String errorMessage = null;
-      if (request.getId() == null
-          || (request.getId().getIntId() == 0 && request.getId().getStringId().isEmpty())) {
-        errorMessage = "Invalid parameter set in GetKeyValuePropertiesRequest.Id";
-      } else if (request.getKeysList().isEmpty() && !request.getGetAll()) {
-        errorMessage =
-            "Keys not found OR getAll flag is false in GetKeyValuePropertiesRequest request";
-      } else if (request.getPropertyName().isEmpty()) {
-        errorMessage = "PropertyName not found in GetKeyValuePropertiesRequest request";
-      }
-
-      if (errorMessage != null) {
-        throw new InvalidArgumentException(errorMessage);
-      }
-
-      List<KeyValueStringProperty> keyValues = metadataDAO.getKeyValueProperties(request);
-      responseObserver.onNext(
-          GetKeyValuePropertiesRequest.Response.newBuilder()
-              .addAllKeyValueProperty(keyValues)
-              .build());
-      responseObserver.onCompleted();
-    } catch (Exception e) {
-      CommonUtils.observeError(
-          responseObserver, e, GetKeyValuePropertiesRequest.Response.getDefaultInstance());
-    }
-  }
-
-  @Override
-  public void deleteKeyValueProperties(
-      DeleteKeyValuePropertiesRequest request,
-      StreamObserver<DeleteKeyValuePropertiesRequest.Response> responseObserver) {
-    try {
-      String errorMessage = null;
-      if (request.getId() == null
-          || (request.getId().getIntId() == 0 && request.getId().getStringId().isEmpty())) {
-        errorMessage = "Invalid parameter set in DeleteKeyValuePropertiesRequest.Id";
-      } else if (request.getKeysList().isEmpty() && !request.getDeleteAll()) {
-        errorMessage =
-            "Keys not found OR deleteAll flag is false in DeleteKeyValuePropertiesRequest request";
-      } else if (request.getPropertyName().isEmpty()) {
-        errorMessage = "PropertyName not found in DeleteKeyValuePropertiesRequest request";
-      }
-
-      if (errorMessage != null) {
-        throw new InvalidArgumentException(errorMessage);
-      }
-
-      metadataDAO.deleteKeyValueProperties(request);
-      responseObserver.onNext(DeleteKeyValuePropertiesRequest.Response.newBuilder().build());
-      responseObserver.onCompleted();
-    } catch (Exception e) {
-      CommonUtils.observeError(
-          responseObserver, e, DeleteKeyValuePropertiesRequest.Response.getDefaultInstance());
-    }
-  }
-
-  public static String createRandomName() {
-    return Nomen.est().adjective().color().animal().withSeparator("-").get();
-  }
-
-  @Override
-  public void generateRandomName(
```

Comment:
Actually we should keep this one! It's used in other parts of the system too. I forgot it was in the metadata service. I was only thinking about labels and attributes.